vm/neovm/value_stack.go

```diff
@@ -56,6 +56,18 @@ func (self *ValueStack) Insert(index int64, t types.VmValue) error {
 	return nil
 }
 
+func (self *ValueStack) PeekNByteArray(index int64) ([]byte, error) {
```

Comment:
rename as `PeekAsBytes` to make it consistent with other function and move to `value_stake_convention.go`